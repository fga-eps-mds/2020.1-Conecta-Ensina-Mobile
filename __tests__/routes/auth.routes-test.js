import React from 'react';
import {render} from '@testing-library/react-native';
import AuthRoutes from '../../src/routes/auth.routes';
import {NavigationContainer} from '@react-navigation/native';

describe('Testing AuthRoutes ', () => {
  test('Should Match Snapshot', () => {
    const route = render(
      <NavigationContainer>
        <AuthRoutes />
      </NavigationContainer>,
    ).toJSON();

    expect(route).toMatchSnapshot();
  });
});
