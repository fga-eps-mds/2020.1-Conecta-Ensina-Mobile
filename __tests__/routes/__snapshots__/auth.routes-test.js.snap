// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing AuthRoutes  Should Match Snapshot 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <RNCSafeAreaProvider
    onInsetsChange={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
        },
        undefined,
      ]
    }
  >
    <View
      onLayout={[Function]}
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        pointerEvents="box-none"
        style={
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <View
          collapsable={false}
          style={
            Object {
              "opacity": 1,
            }
          }
        />
        <View
          accessibilityElementsHidden={false}
          closing={false}
          gestureVelocityImpact={0.3}
          importantForAccessibility="auto"
          onClose={[Function]}
          onGestureBegin={[Function]}
          onGestureCanceled={[Function]}
          onGestureEnd={[Function]}
          onOpen={[Function]}
          onTransitionStart={[Function]}
          pointerEvents="box-none"
          style={
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
          transitionSpec={
            Object {
              "close": Object {
                "animation": "spring",
                "config": Object {
                  "damping": 500,
                  "mass": 3,
                  "overshootClamping": true,
                  "restDisplacementThreshold": 10,
                  "restSpeedThreshold": 10,
                  "stiffness": 1000,
                },
              },
              "open": Object {
                "animation": "spring",
                "config": Object {
                  "damping": 500,
                  "mass": 3,
                  "overshootClamping": true,
                  "restDisplacementThreshold": 10,
                  "restSpeedThreshold": 10,
                  "stiffness": 1000,
                },
              },
            }
          }
        >
          <View
            needsOffscreenAlphaCompositing={false}
            pointerEvents="box-none"
            style={
              Object {
                "flex": 1,
                "marginTop": 0,
              }
            }
          >
            <View
              collapsable={false}
              forwardedRef={[Function]}
              onGestureHandlerEvent={[Function]}
              onGestureHandlerStateChange={[Function]}
              style={
                Object {
                  "flex": 1,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                    Object {
                      "translateX": 0,
                    },
                  ],
                }
              }
            >
              <View
                pointerEvents="none"
                style={
                  Object {
                    "backgroundColor": "rgb(242, 242, 242)",
                    "bottom": 0,
                    "left": 0,
                    "position": "absolute",
                    "shadowColor": "#000",
                    "shadowOffset": Object {
                      "height": 1,
                      "width": -1,
                    },
                    "shadowOpacity": 0.3,
                    "shadowRadius": 5,
                    "top": 0,
                    "width": 3,
                  }
                }
              />
              <View
                style={
                  Array [
                    Object {
                      "flex": 1,
                      "overflow": "hidden",
                    },
                    Array [
                      Object {
                        "backgroundColor": "rgb(242, 242, 242)",
                      },
                      undefined,
                    ],
                  ]
                }
              >
                <View
                  style={
                    Object {
                      "flex": 1,
                      "flexDirection": "column-reverse",
                    }
                  }
                >
                  <View
                    style={
                      Object {
                        "flex": 1,
                      }
                    }
                  >
                    <View
                      style={
                        Array [
                          Object {
                            "alignItems": "center",
                            "backgroundColor": "#184E79",
                            "flexBasis": 0,
                            "flexGrow": 1,
                            "flexShrink": 1,
                            "justifyContent": "center",
                          },
                        ]
                      }
                      testID="background"
                    >
                      <View
                        style={
                          Array [
                            Object {
                              "alignItems": "center",
                              "backgroundColor": "#184E79",
                              "flexBasis": 0,
                              "flexGrow": 0.7,
                              "flexShrink": 1,
                              "justifyContent": "center",
                              "width": "100%",
                            },
                          ]
                        }
                      >
                        <Image
                          source={
                            Object {
                              "testUri": "../../../src/assets/logo.png",
                            }
                          }
                          style={
                            Array [
                              Object {
                                "height": "76.5%",
                                "marginBottom": "14%",
                                "width": "50%",
                              },
                            ]
                          }
                          testID="Logo"
                        />
                      </View>
                      <View
                        style={
                          Array [
                            Object {
                              "alignItems": "center",
                              "backgroundColor": "#ffffff",
                              "borderTopLeftRadius": 20,
                              "borderTopRightRadius": 20,
                              "flexBasis": 0,
                              "flexGrow": 1,
                              "flexShrink": 1,
                              "justifyContent": "center",
                              "width": "100%",
                            },
                          ]
                        }
                      >
                        <View
                          style={
                            Array [
                              Object {
                                "alignItems": "center",
                                "marginBottom": "83.5%",
                              },
                            ]
                          }
                        >
                          <View
                            style={
                              Array [
                                Object {
                                  "alignItems": "center",
                                  "backgroundColor": "#D9D9D9",
                                  "borderRadius": 15,
                                  "height": 360,
                                  "justifyContent": "center",
                                  "marginTop": "55.5%",
                                  "width": 306,
                                },
                              ]
                            }
                          >
                            <View
                              style={
                                Array [
                                  Object {
                                    "alignItems": "center",
                                    "backgroundColor": "#184E79",
                                    "borderRadius": 100,
                                    "height": "16.11%",
                                    "justifyContent": "center",
                                    "marginBottom": 30,
                                    "marginTop": "5%",
                                    "width": "18.95%",
                                  },
                                ]
                              }
                            >
                              <Image
                                source={
                                  Object {
                                    "testUri": "../../../src/assets/user_white.png",
                                  }
                                }
                                style={
                                  Array [
                                    Object {
                                      "height": "63%",
                                      "width": "52%",
                                    },
                                  ]
                                }
                                testID="Icon"
                              />
                            </View>
                            <TextInput
                              allowFontScaling={true}
                              autoCapitalize="none"
                              multiline={true}
                              onChangeText={[Function]}
                              placeholder="Email"
                              placeholderTextColor="#F6F6F6"
                              rejectResponderTermination={true}
                              style={
                                Array [
                                  Object {
                                    "alignItems": "center",
                                    "backgroundColor": "#184E79",
                                    "borderRadius": 10,
                                    "color": "#ffffff",
                                    "height": 45,
                                    "justifyContent": "center",
                                    "marginTop": "5%",
                                    "width": 280,
                                  },
                                ]
                              }
                              textAlign="center"
                              underlineColorAndroid="transparent"
                            />
                            <View
                              style={
                                Array [
                                  Object {
                                    "alignItems": "center",
                                    "backgroundColor": "#184E79",
                                    "borderRadius": 10,
                                    "flexDirection": "row",
                                    "height": 45,
                                    "justifyContent": "center",
                                    "marginTop": "5%",
                                    "width": 280,
                                  },
                                ]
                              }
                            >
                              <View
                                style={
                                  Array [
                                    Object {
                                      "alignItems": "center",
                                      "backgroundColor": "#184E79",
                                      "borderRadius": 10,
                                      "flexDirection": "row",
                                      "height": 45,
                                      "justifyContent": "center",
                                      "marginTop": "5%",
                                      "width": 280,
                                    },
                                  ]
                                }
                              >
                                <TextInput
                                  allowFontScaling={true}
                                  autoCapitalize="none"
                                  onChangeText={[Function]}
                                  placeholder="Senha"
                                  placeholderTextColor="#F6F6F6"
                                  rejectResponderTermination={true}
                                  secureTextEntry={true}
                                  style={
                                    Object {
                                      "color": "#FFFFFF",
                                      "flex": 1,
                                      "fontSize": 14,
                                      "textAlign": "center",
                                    }
                                  }
                                  testID="password"
                                  underlineColorAndroid="transparent"
                                  value=""
                                />
                              </View>
                            </View>
                            <View
                              accessible={true}
                              focusable={true}
                              onClick={[Function]}
                              onResponderGrant={[Function]}
                              onResponderMove={[Function]}
                              onResponderRelease={[Function]}
                              onResponderTerminate={[Function]}
                              onResponderTerminationRequest={[Function]}
                              onStartShouldSetResponder={[Function]}
                              style={
                                Object {
                                  "alignItems": "center",
                                  "backgroundColor": "#184E79",
                                  "borderRadius": 10,
                                  "height": 50,
                                  "justifyContent": "center",
                                  "marginTop": 20,
                                  "opacity": 1,
                                  "width": 200,
                                }
                              }
                              testID="signIn"
                            >
                              <Text
                                medium={true}
                                style={
                                  Array [
                                    Object {
                                      "color": "#ffffff",
                                      "fontFamily": "Roboto",
                                      "fontSize": 25,
                                      "textAlign": "center",
                                    },
                                  ]
                                }
                                white={true}
                              >
                                Entrar
                              </Text>
                            </View>
                            <View
                              accessible={true}
                              focusable={true}
                              onClick={[Function]}
                              onResponderGrant={[Function]}
                              onResponderMove={[Function]}
                              onResponderRelease={[Function]}
                              onResponderTerminate={[Function]}
                              onResponderTerminationRequest={[Function]}
                              onStartShouldSetResponder={[Function]}
                              style={
                                Object {
                                  "marginBottom": "10%",
                                  "marginTop": 10,
                                  "opacity": 1,
                                }
                              }
                              testID="signUp"
                            >
                              <Text
                                style={
                                  Array [
                                    Object {
                                      "color": "#ffffff",
                                      "fontFamily": "Roboto",
                                      "fontSize": 20,
                                      "textShadowColor": "#596C7C",
                                      "textShadowOffset": Object {
                                        "height": 1.5,
                                        "width": -1.5,
                                      },
                                      "textShadowRadius": 4,
                                    },
                                  ]
                                }
                              >
                                Registre-se
                              </Text>
                            </View>
                          </View>
                        </View>
                      </View>
                    </View>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
    <View
      pointerEvents="box-none"
      style={
        Object {
          "height": 64,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
          "zIndex": 1,
        }
      }
    />
  </RNCSafeAreaProvider>
</View>
`;
